index.test.ts

```diff
@@ -0,0 +1,122 @@
+import { processEvent } from ".";
```

Comment:
Please use prettier with the provided `.prettierrc` config file and format the file.